pkg/generators/golang/errors_generator.go

```diff
@@ -245,6 +247,13 @@ func (g *ErrorsGenerator) generateCommonErrors() error {
 			return b
 		}
 
+		// Details sets additional details included in the error.
+		func (b *ErrorBuilder) Details(value []interface{}) *ErrorBuilder {
+			b.details = value
+			b.bitmap_ |= 24
```

Comment:
This number is a bit mask, so it should be a power of two. In this case it should be 32.